### ISSUE TITLE
Errors upgrading to 0.3.0

### ISSUE THREAD

**Issue by hamstu** [EugenMayer/docker-sync#302]
I said yes to the 0.3.0 upgrade prompt for docker-sync, but then when I tried to boot up my containers I got this:

```
$ docker-sync-stack start
     warning  The installation progress of docker-sync 0.3.0 has changed, brew is now mandatory - you need to uninstall docker-sync/unox/unison ! :

_Please_ read :): https://github.com/EugenMayer/docker-sync/wiki/1.3-Upgrade-Guide

          ok  The installation progress of docker-sync 0.3.0 has changed, brew is now mandatory - you need to uninstall docker-sync/unox/unison ! :

_Please_ read :): https://github.com/EugenMayer/docker-sync/wiki/1.3-Upgrade-Guide

I will install reinstall docker-sync for you using the above command (y/N) yes
sh: -c: line 0: unexpected EOF while looking for matching `"'
sh: -c: line 1: syntax error: unexpected end of file
Error: No such keg: /usr/local/Cellar/unox
       error  You install unison-fsmonitor (unox) not using brew. Please uninstall it and run docker-sync again, so we can install it for you
       error  exit
```

I believe this is related to some typos on this line:
https://github.com/EugenMayer/docker-sync/blob/0.3.0/lib/docker-sync/upgrade_check.rb#L70

```
rm -f /usr/loca/bin/unison-fsmonitor
               ^ missing an 'l'
```

And there's also a trailing `"` in the `cmd1` string.

All that said, I tried running the corrected commands manually and I get stuck at the last step:

```
$ brew install eugenmayer/dockersync/dockersync
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 2 taps (caskroom/cask, homebrew/core).
==> Updated Formulae
kobalt

Error: No available formula with the name "eugenmayer/dockersync/dockersync"
==> Searching for a previously deleted formula...
Error: No previously deleted formula found.
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching taps...
Error: No formulae found in taps.
```

I'm on a MacBook Pro / macOS Sierra 10.12.4

**Comment by Nomad145** [comment]
@hamstu, I was able to get back up and running by following the installation guide here: https://github.com/EugenMayer/docker-sync/wiki/1.-Installation

**Comment by hamstu** [comment]
Thanks @Nomad145. I got it working by changing the last step to `brew install eugenmayer/dockersync/unox` (Pretty sure that's it.)

But then nothing was syncing... 😢  – seems like `0.3.0` needs some bugs worked out.

In the interim I've been able to get my stack working again by downgrading; `gem uninstall docker-sync && gem install docker-sync -v 0.2.3`.

**Comment by ignatiusreza** [comment]
> I kept seeing this error in my terminal any time I made a change

That's strange.. that lines supposed to run only once when loading config.. do you by any chances use custom scripting? and could you share your configuration?

**Comment by EugenMayer** [comment]
Beside that, please use the boilerplate to reproduce it so we can have it tracked down faster, thanks

**Comment by EugenMayer** [comment]
@hamstu @Nomad145 beside posting your configs and reproducing with the boilerplate, @ignatiusreza and i found some issues in the unison installation and the legacy unox deinstallation. Please try to install 0.3.2 once again.

I suppose the config being blank might being a YAML issue, so config formatting. Ensure your indentation is right, use http://www.yamllint.com/ to ensure this

**Comment by EugenMayer** [comment]
this was fixed with 0.3.2 / 0.3.5
